cypress/integration/admin_account_test/report_page_layout_test.js

```diff
@@ -0,0 +1,100 @@
+describe("Report Page Layout Test", function(){
+    beforeEach(function(){
+        cy.openLoginDialogue();
+		cy.get(".login-form").should("contain", "Login to BioTestMine");
+		cy.get("input#email").type("test_user@mail_account");
+		cy.get("input[type='password']").type("secret");
+        cy.intercept('POST', '/api/auth/login').as('login');
+		cy.get(".login-form")
+			.find("button")
+			.contains('Login')
+		 	.click();
+        cy.wait('@login');
+        cy.get(".logon.dropdown.success").should("exist").click();
+        cy.get(".logon.dropdown.success").should("contain", "test_user@mail_account"); //flaky
```

Comment:
What do you think about adding a cypress command for logging in as admin? I've seen this snippet used several places.